ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -150,6 +162,10 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
         return options.filter(option => option.optionLabel.toLowerCase().includes(filterValue));
     }
 
+    private get shouldBeSorted(): boolean {
+        return !this.config.picklistsWithNoSorting.includes(this.block.stableId);
```

Comment:
we need to check if picklistsWithNoSorting is defined (will be not defined for most of studies)